app/src/components/search/content.tsx

```diff
@@ -29,44 +42,71 @@ const SearchContent = () => {
       <h2>Search params: {query} </h2>
       <br />
       <Box sx={{ display: "flex", gap: "xs", marginBottom: "l" }}>
-        <Text sx={{ fontWeight: "500", marginBottom: 0, marginTop: "xs" }}>
-          {" "}
-          Sort by{" "}
-        </Text>{" "}
-        <Menu
-          //showSelectionAsLabel
-          showLabel
-          selectedItem="chronological-descending"
-          labelText={"Sort By"}
-          listItemsData={[
-            {
-              id: "chronological-descending",
-              label: "Newest to oldest",
-              onClick: onMenuClick,
-              type: "action",
-            },
-            {
-              id: "chronological-ascending",
-              label: "Oldest to newest",
-              onClick: onMenuClick,
-              type: "action",
-            },
-            {
-              id: "alphabetical-descending",
-              label: "Title A to Z",
-              onClick: onMenuClick,
-              type: "action",
-            },
-            {
-              id: "alphabetical-ascending",
-              label: "Title Z to A",
-              onClick: onMenuClick,
-              type: "action",
-            },
-          ]}
-        />
+        {showFilter ? (
+          <>
+            <Text sx={{ fontWeight: "500", marginBottom: 0, marginTop: "xs" }}>
+              {" "}
+              Sort by{" "}
+            </Text>{" "}
+            <Menu
+              //showSelectionAsLabel
+              showLabel
+              selectedItem="chronological-descending"
+              labelText={"Sort By"}
+              listItemsData={[
+                {
+                  id: "chronological-descending",
+                  label: "Newest to oldest",
+                  onClick: onMenuClick,
+                  type: "action",
+                },
+                {
+                  id: "chronological-ascending",
+                  label: "Oldest to newest",
+                  onClick: onMenuClick,
+                  type: "action",
+                },
+                {
+                  id: "alphabetical-descending",
+                  label: "Title A to Z",
+                  onClick: onMenuClick,
+                  type: "action",
+                },
+                {
+                  id: "alphabetical-ascending",
+                  label: "Title Z to A",
+                  onClick: onMenuClick,
+                  type: "action",
+                },
+              ]}
+            />
+          </>
+        ) : (
+          <></>
```

Comment:
Make this `null` instead of the react fragment. It's the recommended practice from React.